pypeit/scripts/sensfunc.py

```diff
@@ -94,12 +94,12 @@ def parse_args(options=None, return_parser=False):
                         help="Ouput file for sensitivity function. If not specified, the sensitivity function will "
                              "be written out to a standard filename in the current working directory, i.e. if the "
                              "standard spec1d file is named spec1d_b24-Feige66_KASTb_foo.fits the sensfunc will be "
-                             "written to sens_b24-Feige66_KASTb_foo.fits")
+                             "written to sens_b24-Feige66_KASTb_foo.fits. A QA file will also be written as "
```

Comment:
I don't get these default names.  Why `KAST`?

Reply:
This was here before.